nni/compression/pytorch/utils/mask_conflict.py

```diff
@@ -249,6 +252,10 @@ def fix_mask(self):
                         else:
                             channel_masks.append(
                                 (mask.abs().sum(1) != 0).int())
+                    elif type(m).__name__ == 'Embedding':
```

Comment:
should it be ``isinstance(m, nn.Linear)``?

Reply:
yes, I think `isinstance(m, nn.Embedding)` is better, just follow the old code style.